docs/docs/05-reusable-components.md

```diff
@@ -72,7 +72,16 @@ React.createClass({
       if (!/matchme/.test(props[propName])) {
         return new Error('Validation failed!');
       }
-    }
+    },
+
+    // Further, you can also supply a custom type checker to `arrayOf`. The
+    // function will be called on each item in the array, The first two
+    // arguments will be the array itself and the current element's index.
+    customArrayProp: React.PropTypes.arrayOf(function(arr, i, componentName, location, propFullName) {
```

Comment:
Let’s remove `location` and `propFullName` here unless we consistently use it both in this example and in the `customProp` above.